### DISCUSSION TITLE
[Chapter 10] 트랜잭션 격리수준

### DISCUSSION THREAD

**Discussion by coalong** [cowaine/spring-boot-for-msa#169] [Chapter 10. Redis and Spring Cache]
10.5 레디스 분산 락 사용 예제에서 (그림 10-6 밑 p. 666) 다음과 같은 문장이 있다.

> "SERIALIZABLE 격리 수준을 사용하지 않는 한 데이터베이스만으로는 데이터 무결성을 보장할 수 없다."

이 문장으로 설명만 하고 넘어가는데, 
SERIALIZABLE 격리 수준만 가지는 무언가가 있는 건지 궁금합니다!

**Comment by JoisFe** [comment]
https://velog.io/@yujiniii/%EB%8D%B0%EC%9D%B4%ED%84%B0%EB%B2%A0%EC%9D%B4%EC%8A%A4-%ED%8A%B8%EB%9E%9C%EC%9E%AD%EC%85%98-%EA%B2%A9%EB%A6%AC-%EC%88%98%EC%A4%80

**Reply by JoisFe** [reply]
<img width="603" alt="image" src="https://github.com/cowaine/spring-boot-for-msa/assets/90208100/b8ec4c81-4002-465b-85e2-136b19c5c1b4">

**Comment by Irisation23** [comment]
### 금주 Check Point

-`SERIALIZABLE ` 격리 수준은 성능에 문제가 발생할 수 있다.
